frontend/packages/operator-lifecycle-manager/src/utils.ts

```diff
@@ -1,9 +1,8 @@
 import { ClusterServiceVersionKind, CRDDescription } from './types';
 import { referenceForProvidedAPI } from './components';
 
-export const getInternalObjects = (csv: ClusterServiceVersionKind) => {
-  const internals: string =
-    csv?.metadata?.annotations?.['operators.operatorframework.io/internal-objects'];
+export const getInternalObjects = (csv: any, path: string = 'metadata.annotations') => {
+  const internals: string = csv?.[path]?.['operators.operatorframework.io/internal-objects'];
```

Comment:
I don't think this will work as expected when `path` is `metadata.annotations`. You will be trying to access 

```js
{ 
  "metadata.annotations": { 
    "operators.operatorframework.io/internal-objects": ""
  }
}
````

Reply:
Good catch. Yes I need to use lodash.